disnake/types/gateway.py

```diff
@@ -278,6 +278,7 @@ class ReadyEvent(TypedDict):
     user: User
     guilds: List[UnavailableGuild]
     session_id: str
+    resume_gateway_url: str
```

Comment:
is this provided on gateway v9 as well?

Reply:
Yup, seems to be present in all currently available (and deprecated) versions.